src/chai/ManagedArray.hpp

```diff
@@ -91,10 +94,14 @@ class ManagedArray : public CHAICopyable
 public:
   using T_non_const = typename std::remove_const<T>::type;
 
+  CHAI_HOST_DEVICE ManagedArray();
+
   /*!
    * \brief Default constructor creates a ManagedArray with no allocations.
    */
-  CHAI_HOST_DEVICE ManagedArray();
+  CHAI_HOST_DEVICE ManagedArray(
+      std::initializer_list<chai::ExecutionSpace> spaces,
+      std::initializer_list<umpire::Allocator> allocators);
```

Comment:
@trws - I couldn't make these constructors work using double brace enclosed initializer lists with nvcc. Should I be able to make it work?

Reply:
I'm honestly not sure, would have to dig into this further.

Reply:
No problems, thanks for looking.